setup/productionize/persistence/orca-sql.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: single
+title:  "Set up Orca to use SQL"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Orca's execution state is stored in Redis by default, but can be configured for SQL. 
+In this topology, Redis is still required for the work queue.
+Using SQL for execution state will make your Spinnaker installation more durable.
+
+This guide will go over MySQL setup, how to configure Orca, as well as how to perform a zero-downtime migration from Redis to SQL.
+
+If you already have an Orca deployment, you should also refer to the [Redis to SQL Migration Guide](/guides/operator/orca-redis-to-sql/).
+
+## MySQL 5.7 Setup
+
+Orca ships with MySQL drivers by default, but you can include your own JDBC drivers on the classpath if you need to connect to a different database.
+
+Orca has been developed and tested targeting MySQL 5.7. As part of this, setting MySQL's `tx_isolation` value to `READ-COMMITTED` is essential to successfully running Orca in SQL. 
+While Orca will attempt to set this on connection sessions, it is better to have it set on the database itself.
```

Comment:
@robzienert would it be possible to change this to a procedure? 
1. Set `tx_isolation` to `READ-COMMITTED` on your MySQL.
2. Set up the schema and database uses [users?]:
   ```sql
   CREATE SCHEMA `orca`...
   etc.
```
etc.